inst/@sym/sym.m

```diff
@@ -190,239 +189,231 @@
-      % Use S(), as we're not forcing Symbol()
-      assert (isempty (asm))   % sym('pi', 'integer')
-      if (doDecimalCheck && ~isempty(strfind(x, '.')))
-        warning('possibly unintended decimal point in constructor string');
-      end
-      % x is raw sympy, could have various quotes in it
-      cmd = sprintf('z = sympy.S("%s")', strrep(x, '"', '\"'));
-
-    else % useSymbolNotS
-      if (isempty(asm))
-        cmd = sprintf('z = sympy.Symbol("%s")', x);
-
-      elseif (isscalar(asm) && isscalar(asm{1}) && isstruct(asm{1}))
-        % we have an assumptions dict
-        cmd = sprintf('return sympy.Symbol("%s", **_ins[0]),', x);
-        s = python_cmd (cmd, asm{1});
-        return
-
-      elseif (iscell(asm))
-        valid_asm = assumptions('possible');
-        for n=1:length(asm)
-          assert(ischar(asm{n}), 'sym: assumption must be a string')
-          assert(ismember(asm{n}, valid_asm), ...
-                 'sym: that assumption is not supported')
-        end
-        cmd = ['z = sympy.Symbol("' x '"' ...
-               sprintf(', %s=True', asm{:}) ')'];
-      else
-        error('sym: invalid extra input, perhaps invalid assumptions?');
+      %%Check if the user try to execute operations from sym
+      if ~isempty (regexp (xc, '\!|\&|\^|\:|\*|\/|\\|\+|\-|\>|\<|\=|\~'))
+        warning ('Please avoid execute operations from sym function.');
       end
-    end % useSymbolNotS
 
-  else
-    x
-    class(x)
-    nargin
-    error('conversion to symbolic with those arguments not (yet) supported');
+      cmd = {'x = "{s}"'
+             'try:'
+             '    return (0, (0, 0), S(x, rational=True))'
+             'except Exception as e:'
+             '    lis = set()'
+             '    if "(" in x or ")" in x:'
+             '        x2 = split("\(|\)| |,", x)'
+             '        x2 = [p for p in x2 if p]'
+             '        for i in x2:'
+             '            try:'
+             '                if eval("callable(" + i + ")"):'
+             '                    lis.add(i)' 
+             '            except:'
+             '                pass'
+             '    if len(lis) > 0:'
+             '        return (str(e), (1, "" if len(lis) == 1 else "s"), "\", \"".join(str(e) for e in lis))'
+             '    return (str(e), (2, 0), x)' };
+           
+      [err flag s] = python_cmd (strrep (cmd, '{s}', x));
+      switch flag{1}
+        case 1  %%Bad call to python function
+          disp (['Python: ' err]);
+          disp (['error: Error using the "' s '" Python function' flag{2} ', you wrote it correctly?']);
+          error ('if this do not was intentional please use other var name.');
+        case 2  %%Something else
+          disp (['Python: ' err]);
+          error (['You can not use var name "' s '" for a error, if is a bug please report it.']);
+      end
+      return
+    end
   end
 
-  s = python_cmd ({cmd 'return z,'});
+  error ('Input not supported yet.')
```

Comment:
Although this is ok too but how about the original text `conversion to symbolic with those arguments not (yet) supported`?